assets/src/onboarding-wizard/index.js

```diff
@@ -49,8 +50,13 @@ const { ajaxurl: wpAjaxUrl } = global;
 export function Providers( { children } ) {
 	return (
 		<ErrorContextProvider>
-			<ErrorBoundary exitLink={ FINISH_LINK } fullScreen={ true }>
-
+			<ErrorBoundary
+				exitLink={ {
```

Comment:
The prop type for `exitLink` is a `string`, but you're passing an object here.

Reply:
Good catch. In fact, after addressing your feedback on the `finishLink` prop, this becomes outdated.